src/thing/thing.test.ts

```diff
@@ -1422,25 +1432,35 @@ describe("thingAsMarkdown", () => {
         someLiteral
       )
     );
-    thingWithRdfValues.add(
-      DataFactory.quad(
-        thingWithRdfValues.internal_localSubject,
-        DataFactory.namedNode("https://some.vocab/predicate"),
-        DataFactory.quad(
-          DataFactory.blankNode("Arbitrary Subject in an RDF* nested Quad."),
-          DataFactory.namedNode("https://some.vocab/predicate"),
-          DataFactory.literal("Arbitrary Object in an RDF* nested Quad.")
-        )
-      )
-    );
+
+    // TODO: Indexed RDF/JS implementation needs to be updated to handle RDF*
+    //  types before we can add this test back.
+    // Just to maintain code coverage until the Indexed RDF/JS updates to
+    // include RDF* types...
+    expect(
+      internal_getReadableValue({ termType: "Quad" } as Quad_Object)
+    ).toEqual("??? (nested RDF* Quad)");
+    // thingWithRdfValues.add(
+    //   DataFactory.quad(
+    //     thingWithRdfValues.internal_localSubject,
+    //     DataFactory.namedNode("https://some.vocab/predicate"),
+    //     DataFactory.quad(
+    //       DataFactory.blankNode("Arbitrary Subject in an RDF* nested Quad."),
+    //       DataFactory.namedNode("https://some.vocab/predicate"),
+    //       DataFactory.literal("Arbitrary Object in an RDF* nested Quad.")
+    //     )
+    //   )
+    // );
```

Comment:
That's probably the reason of the mismatch above. Either way, since this PR doesn't introduce `-indexed`, not relevant here.
```suggestion
    thingWithRdfValues.add(
      DataFactory.quad(
        thingWithRdfValues.internal_localSubject,
        DataFactory.namedNode("https://some.vocab/predicate"),
        DataFactory.quad(
          DataFactory.blankNode("Arbitrary Subject in an RDF* nested Quad."),
          DataFactory.namedNode("https://some.vocab/predicate"),
          DataFactory.literal("Arbitrary Object in an RDF* nested Quad.")
        )
      )
    );
```

Reply:
Done - 8f702c67